### PR TITLE
fix(templatesUsePublicRule): Pass member name back through error

### LINKED ISSUE #229: Better logging of errors required
With the new package we get errors like 'You can bind only to public class members.' with occur to happen at line 1, column 1 of the HTML template. But the template has `<div class="main-menu-button"> `on the first line. 

If I look to the code of the templatesUsePublicRule the specific member is known. So could we at least have the name of the member or method that is causing the error in the error message and not a generic error?

### PR DESCRIPTION
**Pull request by comfroels**
Pass the class member name to the error for more clarification

fixes #229

### CODE REVIEW

File: src/templatesUsePublicRule.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ class SymbolAccessValidator extends RecursiveAngularExpressionVisitor {
         .some(m => m.kind === SyntaxKind.current().PrivateKeyword || m.kind === SyntaxKind.current().ProtectedKeyword);
       const width = ast.name.length;
       if (!isPublic) {
-        const failureString = 'You can bind only to public class members.';
+        const failureString = `You can bind only to public class members. ${member.name.getText()} is not a public class member.`;
```
Comment (mgechev) [inline src/templatesUsePublicRule.ts]: Would you surround it in quotes in order to be consistent with other rules?

**Comment by mgechev** [comment]
@comfroels awesome job! Thanks! I left only a single comment.

**Comment by comfroels** [comment]
Yeah not a problem! One of my first OSS contributions.

**Comment by mgechev** [comment]
Thanks again! I'll update the list of contributors in the next few days.

**Pull request merged**